app/(gcforms)/[locale]/(form administration)/form-builder/[id]/preview/Preview.tsx

```diff
@@ -78,9 +78,14 @@ export const Preview = ({
     }));
 
   const { t } = useTranslation(["common", "form-builder", "form-closed"]);
+
   const language = translationLanguagePriority;
   const currentForm = getRenderedForm(formRecord, language);
 
+  const translatedT = useTranslation(["common", "form-builder", "form-closed"], {
```

Comment:
Thinking we should move this up under the other `{t}`

At a glance it's easy to miss that we now have t and translatedT

Do you think the array we pass on will ever be different between the 2 ?  Might want to pull out to a const and pass in the same to both?

Reply:
Nope! It should always be the same, so updated with your suggestions. :)